File: skeleton/submissions/accepted/ac.cpp
#include <iostream>

using namespace std;

int main() {
  int n, ans = -(int)1e9;
  cin >> n;
  for (int i = 0; i < n; i++) {
    int a;
    cin >> a;
    ans = max(ans, a);
  }
  cout << ans << endl;
  return 0;
}

File: skeleton/submissions/accepted/ac.java
import java.io.*;
import java.util.*;

public class ac {
  public static void main(String[] args) throws IOException {
    Scanner sc = new Scanner(System.in);
    int n = sc.nextInt(), ans = -Integer.MAX_VALUE;
    for (int i = 0; i < n; i++) {
      int x = sc.nextInt();
      ans = Math.max(ans, x);
    }
    System.out.println(ans);
  }
}
